tests/js/server/aql/aql-queries-optimizer-sort.js

```diff
@@ -28,52 +26,44 @@
 /// @author Copyright 2012, triAGENS GmbH, Cologne, Germany
 ////////////////////////////////////////////////////////////////////////////////
 
-var jsunity = require("jsunity");
-var internal = require("internal");
-var helper = require("@arangodb/aql-helper");
-var isEqual = helper.isEqual;
-var findExecutionNodes = helper.findExecutionNodes;
-var getQueryResults = helper.getQueryResults;
-var db = require("internal").db;
+const jsunity = require("jsunity");
+const internal = require("internal");
+const helper = require("@arangodb/aql-helper");
+const isEqual = helper.isEqual;
+const findExecutionNodes = helper.findExecutionNodes;
+const getQueryResults = helper.getQueryResults;
+const db = require("internal").db;
 
 ////////////////////////////////////////////////////////////////////////////////
 /// @brief test suite
 ////////////////////////////////////////////////////////////////////////////////
 
 function ahuacatlQueryOptimizerSortTestSuite () {
-  var collection = null;
-  var cn = "UnitTestsAhuacatlOptimizerSort";
-  var idx = null;
+  const cn = "UnitTestsAhuacatlOptimizerSort";
```

Comment:
Ahuacatl?

Reply:
It's just a name. It doesn't really matter IMO.